src/PageView/WorkoutsMain/WorkoutList/WorkoutListComponent.css

```diff
@@ -29,6 +29,17 @@
 
 @media (min-width: 600px) {
     .containerBottom {
-        grid-template-columns: repeat(3, 1fr);
+        display: grid;
+        grid-template-columns: 1fr 1fr;;
```

Comment:
remove the extra ; at the end of the line

Reply:
cheers mate